docs/src/content/docs/de/reference/frontmatter.md

```diff
@@ -311,14 +351,15 @@ sidebar:
 
 #### `badge`
 
-**type:** <code>string | <a href="/de/reference/configuration/#badgeconfig">BadgeConfig</a></code>
+**Typ:** <code>string | <a href="/de/reference/configuration/#badgeconfig">BadgeConfig</a></code>
 
 Füge der Seite in der Seitenleiste ein Abzeichen hinzu, wenn es in einer automatisch generierten Gruppe von Links angezeigt wird.
 Bei Verwendung einer Zeichenkette wird das Abzeichen mit einer Standard-Akzentfarbe angezeigt.
 Optional kann ein [`BadgeConfig` Objekt](/de/reference/configuration/#badgeconfig) mit den Feldern `text` und `variant` übergeben werden, um das Abzeichen anzupassen.
```

Comment:
This sentence seems to be missing the changes from this [PR](https://github.com/withastro/starlight/pull/1530/files#diff-6d478912950527786985d16b5fef998fc9a98ffc282f20ebe557b822d0e717bd).